### PR TITLE
UIOAIPMH-8: add ability for duplicate set

### PR DESCRIPTION
**Pull request by Dmitriy-Litvinenko**
# Purpose
Duplicate OAI-PMH set
# Link
https://issues.folio.org/browse/UIOAIPMH-8

### CODE REVIEW

File: test/bigtest/tests/sets/view-test.js [hunk 1]
```diff
@@ -56,10 +56,34 @@ describe('Sets', () => {
           expect(SetsViewInteractor.editButton.isPresent).to.be.true;
         });
 
+        it('should has duplicate action button', () => {
```
Comment (ViktorSoroka) [inline test/bigtest/tests/sets/view-test.js:59]:
```suggestion
        it('should have duplicate action button', () => {
```

File: test/bigtest/tests/sets/duplicate-test.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  it,
+} from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import setupApplication from '../../helpers/setup-application';
+import SetsFormInteractor from '../../interactors/sets-form';
+import translation from '../../../../translations/ui-oai-pmh/en';
+
+describe('Sets', () => {
+  describe('Duplicate', () => {
```
Comment (ViktorSoroka) [inline test/bigtest/tests/sets/duplicate-test.js:13]: Seems like this one can be removed.

File: test/bigtest/tests/sets/duplicate-test.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      let setsTest;
+
+      beforeEach(function () {
+        setsTest = this.server.create('set');
+
+        this.visit(`/settings/oai-pmh/sets/${setsTest.id}/duplicate`);
+      });
+
+      describe('Pane header', () => {
+        it('should be correct title', () => {
```
Comment (ViktorSoroka) [inline test/bigtest/tests/sets/duplicate-test.js:25]:
```suggestion
        it('should render correct title', () => {
```

File: test/bigtest/tests/sets/duplicate-test.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      setupApplication();
+      let setsTest;
+
+      beforeEach(function () {
+        setsTest = this.server.create('set');
+
+        this.visit(`/settings/oai-pmh/sets/${setsTest.id}/duplicate`);
+      });
+
+      describe('Pane header', () => {
```
Comment (ViktorSoroka) [inline test/bigtest/tests/sets/duplicate-test.js:24]: Not sure that describe is needed here.

Comment (Dmitriy-Litvinenko) [reply test/bigtest/tests/sets/duplicate-test.js:24]: Removed

File: test/bigtest/tests/sets/duplicate-test.js (hunk 3 above)
Comment (Dmitriy-Litvinenko) [reply test/bigtest/tests/sets/duplicate-test.js:25]: Fixed

File: test/bigtest/tests/sets/view-test.js (hunk 1 above)
Comment (Dmitriy-Litvinenko) [reply test/bigtest/tests/sets/view-test.js:59]: Fixed

**Pull request merged**